src/js/script.js

```diff
@@ -0,0 +1,63 @@
+let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, ''];
+let indexOfVoid = 15;
+let winCombination1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+let winCombination2 = [1, 5, 9, 13, 2, 6, 10, 14, 3, 7, 11, 15, 4, 8, 12];
+
+function shuffle() {
+    let mySet = new Set();
+
+    function randomInteger(min, max) {
+        let rand = min + Math.random() * (max + 1 - min);
+        rand = Math.floor(rand);
+        return rand;
+    }
+
+    while (mySet.size < 15) {
```

Comment:
Look at this screenshot :)
![image](https://user-images.githubusercontent.com/6029583/54091583-5a3ce980-438a-11e9-85e8-9c453eac1a2e.png)

Try to avoid hardcode use length of your array